src/components/BackLink/__tests__/Backlink.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { withAppContext } from 'test/utils';
+import BackLink from '..';
+
+describe('src/components/BackLink', () => {
+  it('should render correctly', () => {
+    const label = 'I am a backlink';
+    const to = '/foo/bar/baz';
+    const { container, getByText } = render(
+      withAppContext(<BackLink to={to}>{label}</BackLink>)
+    );
+
+    expect(getByText(label)).toBeInTheDocument();
+
+    const link = container.querySelector('a');
+    expect(link).toBeInTheDocument();
+    expect(link.getAttribute('href')).toEqual(to);
+  });
```

Comment:
there is a test missing for clicking the link

Reply:
What should the test verify in your opinion? The element is an anchor; there is no need that the browser does what it is supposed to be doing.

Reply:
the link should link to the right place that is part of the component

Reply:
we should have a talk about the with Carlos about what to test and what not in general